index.html

```diff
@@ -8,20 +8,40 @@
     <link rel="stylesheet" href="style.css" />
     <title>Awsome Books</title>
   </head>
-  <body class="background-image">
-    <main>
-      <section class="hero is-medium">
+  <body class="background-image" id="body">
+    <header class="is-dark">
+      <nav class="navbar is-dark h-p">
+        <div class="container">
+          <div class="navbar-brand">
+            <a class="navbar-item is-size-4" href="#" id="index"> Awesome Book Store </a>
+            <a role="button" class="navbar-burger" data-target="navbarBasicExample">
+              <span aria-hidden="true"></span>
```

Comment:
- [ ] It seems that you have forgotten to display the current date below the header as mentioned in the project requirements below 👇 
![image](https://user-images.githubusercontent.com/55465133/138325121-f6e47fda-396b-4e22-85ff-f2216e72e494.png)
To display the date you will need to use the library [Luxon](https://moment.github.io/luxon/docs/manual/install.html)
 [Get the current date and time](https://moment.github.io/luxon/docs/manual/tour.html#get-the-current-date-and-time)